makefile

```diff
@@ -174,6 +174,8 @@ ifdef TARGET
     else ifneq ($(filter msdos%,$(triplet)),)
       TARGET_OS := dos
     else ifneq ($(filter freebsd%,$(triplet)),)
+      TARGET_OS := dragonfly
+    else ifneq ($(filter dragonfly%,$(triplet)),)
```

Comment:
These two lines should be swapped, otherwise FreeBSD and DragonFly are mixed up, which was probably not intended here. I can do that.